### PR TITLE
Update plot.rst

### PR DESCRIPTION
**Pull request by CovMat**
cpt设置多边形填充色

### CODE REVIEW

File: source/module/plot.rst [hunk 1]
```diff
@@ -42,13 +42,13 @@ plot
 **-C**\ *cpt*
     指定CPT文件或颜色列表
 
-    该选项后跟一个CPT文件名，绘制出的符号或线段的颜色由Z值决定。
+    该选项后跟一个CPT文件名，绘制出的线条颜色，或符号、多边形的填充颜色由Z值决定。
```
Comment (wangliang1989) [inline source/module/plot.rst:45]:
```suggestion
    该选项后跟一个CPT文件名，使线条的颜色或符号、多边形的填充颜色由 Z 值决定。
```

File: source/module/plot.rst [hunk 2]
```diff
@@ -136,8 +136,10 @@ plot
 
     在不使用 ``-F`` 选项的情况下，默认值为 ``-Fcs`` 。该选项的具体示例在后面给出。
 
-**-G**\ *fill*
+**-G**\ *fill*\|\ **+z** :ref:`(more ...) <-Gfill_attrib>`
     设置符号或多边形的填充色。多段数据中数据段头记录中的 ``-G`` 选项会覆盖命令行中的设置。
+    如果在多段记录的头段中设定了Z值，或是使用了 ``-Z`` 选项设定了Z值，使用 **-G+z** 将会
```
Comment (wangliang1989) [inline source/module/plot.rst:141]:
```suggestion
    如果在多段记录的头段中设定了 Z 值，或是使用了 ``-Z`` 选项设定了Z值，使用 **-G+z** 将会
```

Comment (CovMat) [reply source/module/plot.rst]: done

**Comment by seisman** [comment]
@CovMat After someone makes some suggestions, you don't have to manually make the changes on your computer. Instead, you can click the "Add suggestion to batch" button if you agree with the changes and click the "Commit changes" button to commit the changes. 

image

It will save you some time and also give more credits to people who make suggestions.

**Comment by CovMat** [comment]
> @CovMat After someone makes some suggestions, you don't have to manually make the changes on your computer. Instead, you can click the "Add suggestion to batch" button if you agree with the changes and click the "Commit changes" button to commit the changes.
> 
> image
> 
> It will save you some time and also give more credits to people who make suggestions.

原来如此，学到了

File: source/module/plot.rst [hunk 3]
```diff
@@ -136,8 +136,10 @@ plot
 
     在不使用 ``-F`` 选项的情况下，默认值为 ``-Fcs`` 。该选项的具体示例在后面给出。
 
-**-G**\ *fill*
+**-G**\ *fill*\|\ **+z** :ref:`(more ...) <-Gfill_attrib>`
```
Comment (seisman) [inline source/module/plot.rst:139]: The "(more ...)" is linked to the official GMT documentation. I don't think this is what we'd like to see.

Comment (CovMat) [reply source/module/plot.rst]: sure. fixed

File: source/module/plot.rst [hunk 4]
```diff
@@ -42,13 +42,13 @@ plot
 **-C**\ *cpt*
     指定CPT文件或颜色列表
 
-    该选项后跟一个CPT文件名，绘制出的符号或线段的颜色由Z值决定。
+    该选项后跟一个CPT文件名，使线条的颜色或符号、多边形的填充颜色由 Z 值决定。
```
Comment (core-man) [inline source/module/plot.rst:45]: 建议按照线条和填充颜色分类，并修改列表的表述
- 线条、多边形的线条颜色
- 符号和多边形的填充颜色

Comment (CovMat) [reply source/module/plot.rst:45]: 这么分类感觉不行吧。符号和其他两个的数据格式是不一样的。多边形的填充色和线条的颜色数据格式是一样的。

Comment (core-man) [inline source/module/plot.rst:45]:
```suggestion
    该选项后跟一个CPT文件名，使符号和多边形的填充颜色、线段和多边形的线条颜色由 Z 值决定。
```

File: source/module/plot.rst [hunk 5]
```diff
@@ -42,13 +42,13 @@ plot
...
 
     #. 若绘制符号（即使用 ``-S`` 选项），则符号的填充色由数据的第三列Z值决定，
-       其他数据列依次后移一列
+       其他数据列依次后移一列。
     #. 若绘制线段或多边形（即未使用 ``-S`` 选项），则需要在多段数据的数据段头记录中指定
        ``-Z<val>`` (参见 :ref:`table_ascii_id4` )。CPT文件中 ``<val>`` 所对应的颜色，
```
Comment (core-man) [inline source/module/plot.rst:50]:
```suggestion
       ``-Z<val>`` (参见 :ref:`table_ascii_id4` )，CPT文件中 ``<val>`` 所对应的颜色，
```

File: source/module/plot.rst [hunk 6]
```diff
@@ -136,8 +136,10 @@ plot
 
     在不使用 ``-F`` 选项的情况下，默认值为 ``-Fcs`` 。该选项的具体示例在后面给出。
 
-**-G**\ *fill*
+**-G**\ *fill*\|\ **+z** 
     设置符号或多边形的填充色。多段数据中数据段头记录中的 ``-G`` 选项会覆盖命令行中的设置。
+    如果在多段记录的头段中设定了 Z 值，或是使用了 ``-Z`` 选项设定了Z值，使用 **-G+z** 将会
+    从 **-C**\ *cpt* 中读取Z值对应的颜色进行填充。
```
Comment (core-man) [inline source/module/plot.rst:142]: I am a little confused. See official documentation: https://docs.generic-mapping-tools.org/latest/plot.html#g

Comment (CovMat) [reply source/module/plot.rst]: 这边确实是我理解错了。我测试过，设置多边形的填充颜色由cpt控制，只要设置-C和-L就可以，不需要-G。

-G+z的用法暂时没搞懂，先放一边吧，我把+z部分删去，先把前面的笔误和难以理解的部分改过来

**Review by seisman — APPROVED** [review]

**Pull request merged**